### ISSUE TITLE
systemd's service file should use the 'process' KillMode option on Debian also

### ISSUE THREAD

**Issue by douardda** [saltstack/salt#29295]
so one can manage the salt-minion daemon from salt without sawing the branch on which we sit (means: killing the forked salt-minion responsible for handling the command).

See  https://github.com/saltstack/salt/issues/7997#issuecomment-160913751

**Comment by douardda** [comment]
using this setup, on debian jessie, I can run:

```
root@pw01:/srv/salt# salt pd01 pkg.install pkgs="['salt-common','salt-minion']" 
pd01:
    ----------
    python-croniter:
        ----------
        new:
            0.3.4-1
        old:
    python-tz:
        ----------
        new:
            2012c+dfsg-0.1
        old:
    salt-common:
        ----------
        new:
            2015.5.3+ds-1~bpo8+1
        old:
            2015.5.2+ds-1~bpo8+1
    salt-minion:
        ----------
        new:
            2015.5.3+ds-1~bpo8+1
        old:
            2015.5.2+ds-1~bpo8+1

root@pw01:/srv/salt# salt pd01 grains.item saltversion 
pd01:
    ----------
    saltversion:
        2015.5.3

```

**Comment by douardda** [comment]
Ok this has been included in 2014.7.6... but NOT for debian systems... 

why on earth such a fix has been included for all distros bur debian???

**Comment by douardda** [comment]
Debian packaging being handled directly by debian, I close this ticket.
